UnityProject/Assets/Scripts/Objects/Machines/Microwave.cs

```diff
@@ -479,7 +522,8 @@ public MicrowaveRunning(Microwave microwave)
 				StateMsgForExamine = "running";
 				microwave.OnSyncScreenGlow(microwave.screenGlowEnabled, true);
 				microwave.OnSyncOvenGlow(microwave.ovenGlowEnabled, true);
-				microwave.spriteHandler.ChangeSprite(2);
+				microwave.spriteHandler.ChangeSprite((int) SpriteState.Running);
+				microwave.SetWattage(850);
```

Comment:
maybe change this to a  field you can edit in the editor

Reply:
Ah, could do. You want one for all states or just the running one?

Reply:
might as well all states